package.json

```diff
@@ -17,29 +17,31 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "@companieshouse/api-sdk-node": "^2.0.35",
+    "@companieshouse/api-sdk-node": "^2.0.103",
     "@companieshouse/node-session-handler": "^4.1.9",
-    "ch-structured-logging": "git+ssh://git@github.com/companieshouse/ch-structured-logging-node.git#15a811238ceb58988ab755229a5c07b9500200f2",
+    "@companieshouse/structured-logging-node": "^1.0.8",
     "cookie-parser": "1.4.5",
     "escape-html": "1.0.3",
-    "express": "4.17.1",
-    "express-validator": "^6.14.0",
-    "govuk-frontend": "3.11.0",
-    "http-errors": "^1.7.3",
-    "ioredis": "4.16.2",
-    "js-yaml": "^3.14.0",
-    "nunjucks": "3.2.1",
+    "express": "~4.18.2",
+    "express-actuator": "^1.8.4",
+    "express-validator": "^7.0.1",
+    "govuk-frontend": "^4.7.0",
```

Comment:
Just to be aware this may change the look and feel of the application that could be flagged during test. Not a blocker and I recommend going ahead with using this version but in `search.web.ch.gov.uk` this was recently reverted due to it changing the behaviour of the application.